health-services/project/pom.xml

```diff
@@ -114,6 +114,12 @@
             <version>2.13.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.egov.services</groupId>
+            <artifactId>digit-models</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
These are removed fom our Dev branch already, take help from Shubang and remove it